### PR TITLE
docs: getting started

### PR DESCRIPTION
**Pull request by jjmachan**

### CODE REVIEW

File: docs/getstarted/monitoring.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
 
-The Ragas metrics can also be used with other LLM observability tools like
-[Langsmith](https://www.langchain.com/langsmith) and
-[Langfuse](https://langfuse.com/) to get model-based feedback about various
-aspects of you application like those mentioned below
+Additionally, you can use the RAG metrics with other Machine Learning Model (MLM) observability tools like
+- [Langsmith](../howtos/integrations/langsmith.ipynb)
+- [Phoenix (Arize)](https://github.com/Arize-ai/phoenix)
+- [Langfuse](../howtos/integrations/langfuse.ipynb)
+- [OpenLayer](https://openlayer.com/)
```
Comment (jjmachan) [inline docs/getstarted/monitoring.md:20]: are there anymore we can add to this list - ideally we can point them to our integrations page

**Comment by jjmachan** [comment]
added emojies 
image

is that too much?

File: docs/getstarted/testset_generation.md [hunk 2]
```diff
@@ -21,21 +21,21 @@ documents = loader.load()
...
+Here's an example of how to do this:
 
 ```{code-block} python
 for document in documents:
     document.metadata['file_name'] = document.metadata['source']
 ```
 :::
 
-At this point, we have a set of documents at our disposal, which will serve as the basis for creating synthetic Question/Context/Answer triplets.
+At this stage, we have a set of documents ready, which will be used as the foundation for creating synthetic `Question/Context/Answer/Ground_Truth` samples.
```
Comment (shahules786) [inline docs/getstarted/testset_generation.md:34]: We are not creating `Answer` using synthetic data generation
change to 

> Question/Context/ground_truth triplets

File: docs/getstarted/monitoring.md [hunk 3]
```diff
@@ -1,27 +1,29 @@
 (get-started-monitoring)=
-# Monitoring
+# Monitor your RAG in Production
 
-Maintaining the quality and performance of an LLM application in a production environment can be challenging. Ragas provides with basic building blocks that you can use for production quality monitoring, offering valuable insights into your application's performance. This is achieved by constructing custom, smaller, more cost-effective, and faster models.
+Maintaining the quality and performance of a RAG application in a production environment can be challenging. RAG currently provides the essential building blocks that you can use for production-quality monitoring, offering valuable insights into your application's performance. However, we are also working towards building a more advanced production monitoring solution by addressing three questions:
+
+1. How can we ensure the distribution of your production dataset remains consistent with your test set?
```
Comment (shahules786) [inline docs/getstarted/monitoring.md:6]: "How can we" is asking question to the reader, instead I would use "How to"

File: docs/getstarted/evaluation.md [hunk 4]
```diff
@@ -1,36 +1,30 @@
 (get-started-evaluation)=
-# Evaluation
+# Evaluating Your Test Set
```
Comment (shahules786) [inline docs/getstarted/evaluation.md:2]: - "Evaluate your test set" means evaluating quality of test set, I would rephrase to "Evaluate *using* your test set"
- Also change in other instances where similar phrase is used.

File: docs/getstarted/evaluation.md [hunk 5]
```diff
@@ -1,36 +1,30 @@
...
 
-Welcome to the ragas quickstart. We're going to get you up and running with ragas as quickly as you can so that you can go back to improving your Retrieval Augmented Generation pipelines while this library makes sure your changes are improving your entire pipeline.
+Once your test set is ready (whether you've created your own or used the [synthetic test set generation module](get-started-testset-generation)), it's time to evaluate your RAG pipeline. Our aim is to help you set up with Ragas as quickly as possible so that you can focus on enhancing your Retrieval Augmented Generation pipelines while this library ensures your changes are improving the entire pipeline.
```
Comment (shahules786) [inline docs/getstarted/evaluation.md:4]: Is this a mistake?
> while this library ensures your changes are improving the entire pipeline.

File: docs/getstarted/evaluation.md [hunk 6]
```diff
@@ -1,36 +1,30 @@
...
 - question: `list[str]` - These are the questions your RAG pipeline will be evaluated on.
-- answer: `list[str]` - The answer generated from the RAG pipeline and given to the user.
+- answer: `list[str]` - The answer generated from the RAG pipeline and provided to the user.
 - contexts: `list[list[str]]` - The contexts which were passed into the LLM to answer the question.
-- ground_truths: `list[list[str]]` - The ground truth answer to the questions. (only required if you are using context_recall)
-
-Ideally your list of questions should reflect the questions your users give, including those that you have been problematic in the past.
+- ground_truth: `list[str]` - The ground truth answer to the questions.
 
+Ideally, your list of questions should reflect the questions your users ask, including those that have been problematic in the past.
```
Comment (shahules786) [inline docs/getstarted/evaluation.md:27]: > Ideally, your list of questions should reflect the questions your users ask, including those that have been problematic in the past.

Wbt
An ideal test data set should contain samples as close as the ones expected in your real-world use case.

**Review by shahules786 — CHANGES_REQUESTED** [review]

File: docs/getstarted/monitoring.md (hunk 1 above)
Comment (shahules786) [reply docs/getstarted/monitoring.md]: yes, pointing to the integration page would be ideal

File: docs/getstarted/evaluation.md (hunk 5 above)
Comment (jjmachan) [reply docs/getstarted/evaluation.md:4]: nope, its correct i think

File: docs/alfred.py [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+async def fix_doc_with_llm(doc: File, llm: BaseChatModel) -> File:
+    prompt = """\
+fix the following grammar and spelling mistakes in the following text. 
+Please keep the markdown format intact when reformating it. 
+Do not make any change to the parts of text that are for formating or additional metadata for the core text in markdown.
+The target audience for this is developers so keep the tone serious and to the point without any marketing terms. 
+The output text should me in .md format. 
+
+text: {text}
+"""
```
Comment (jjmachan) [inline docs/alfred.py:37]: added a script that helps review docs too

Comment (jjmachan) [reply docs/alfred.py:37]: I've noticed that gpt4 actually helps a lot in keeping the tone professional and consistent.

Comment (jjmachan) [reply docs/alfred.py:37]: but I don't think the prompt is proper - can be improved

**Review by shahules786 — APPROVED** [review]
